projects/step-core/src/lib/modules/entity/types/select-entity-context.interface.ts

```diff
@@ -2,6 +2,6 @@ import { LegacyTableHandle } from '../../table/table.module';
 
 export interface SelectEntityContext extends LegacyTableHandle {
   readonly multipleSelection?: boolean;
-  handleSelect?(selectedId: string): void;
+  handleSelect?(selectedId: any): void;
```

Comment:
any

Reply:
the generic select component we used to have, was based on id, which was always string. I changed it to allow the initiator to receive the entire entity that was selected (like a select). Since it's generic, I don't know how to specify any time